owtf/api/handlers/api_token.py

```diff
@@ -50,7 +50,8 @@ def get(self):
         api_key = str(uuid4())
         try:
             token = self.request.headers.get("Authorization").split()[1]
-            payload = jwt.decode(token, JWT_SECRET_KEY, options=JWT_OPTIONS, algorithms=[JWT_ALGORITHM])
+            payload = jwt.decode(token, JWT_SECRET_KEY, options=JWT_OPTIONS,algorithms=[JWT_ALGORITHM])
```

Comment:
keep the spacing consistent and the last lines on the files? It might be a setting in your editor :)